raymon/tests/profiling/test_intensity.py

```diff
@@ -0,0 +1,31 @@
+from raymon.profiling.extractors.vision import AvgIntensity
+from raymon.tests.conftest import load_data
+
+
+def test_extract(load_data):
+    assert isinstance(AvgIntensity().extract(load_data[0]), float)
+
+
+def test_extract_multiple(load_data):
+    avg_intensity_list = AvgIntensity().extract_multiple(load_data)
+    assert isinstance(avg_intensity_list, list)
+    assert isinstance(avg_intensity_list[0], float)
+
+
+def test_to_jcr():
+    jcr = AvgIntensity().to_jcr()
+    assert jcr["class"] == "raymon.profiling.extractors.vision.intensity.AvgIntensity"
+    assert len(jcr["state"]) == 0
+
+
+def test_from_jcr():
+    extractor = AvgIntensity()
+    assert isinstance(extractor.from_jcr(extractor.to_jcr()), AvgIntensity)
```

Comment:
so, regarding the previous comment, this would be:
`extractor.from_jcr(extractor.to_jcr()["state"])`

Reply:
Why is the "state" not required here? Did you change something?